### PR TITLE
Changed binning in TPC and TOF, fixed histograms in QAResults

### PR DESCRIPTION
**Pull request by zchochul**

### CODE REVIEW

File: PWGCF/FemtoWorld/Core/FemtoWorldPhiSelection.h [hunk 1]
```diff
@@ -501,6 +506,26 @@ void FemtoWorldPhiSelection::fillPhiQA(C const& col, V const& v0, T const& posTr
...
+  }
+
+  // const float invMassPhi = v0.mPhi();
+
+  mHistogramRegistry->fill(HIST("PhiQA/hInvMassPhiNoCuts"), MassPhi);
+
+  /*
+  if (MassPhi > ConfInvMassLowLimit and MassPhi < ConfInvMassUpLimit) {
+    mHistogramRegistry->fill(HIST("PhiQA/hInvMassPhiInvMassCut"), MassPhi);
+  */
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Core/FemtoWorldPhiSelection.h:526]: Please, don't leave code commented

File: PWGCF/FemtoWorld/Core/FemtoWorldPhiSelection.h [hunk 2]
```diff
@@ -27,6 +27,8 @@
 #include "Common/Core/RecoDecay.h"
 #include "Framework/HistogramRegistry.h"
 #include "TLorentzVector.h"
+#include <string>
+#include <vector>
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Core/FemtoWorldPhiSelection.h:31]: MegaLinter requires you to put C style headers, i.e <...> before any other so put them above line 22

File: PWGCF/FemtoWorld/Core/FemtoWorldParticleHisto.h [hunk 3]
```diff
@@ -14,11 +14,12 @@
...
 /// \author Zuzanna Chochulska, WUT Warsaw, zchochul@cern.ch
 
-#ifndef FEMTOWORLDPARTICLEHISTO_H_
-#define FEMTOWORLDPARTICLEHISTO_H_
+#ifndef PWGCF_FEMTOWORLD_CORE_FEMTOWORLDPARTICLEHISTO_H_
+#define PWGCF_FEMTOWORLD_CORE_FEMTOWORLDPARTICLEHISTO_H_
 
 #include "PWGCF/FemtoWorld/DataModel/FemtoWorldDerived.h"
 #include "Framework/HistogramRegistry.h"
+#include <string>
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Core/FemtoWorldParticleHisto.h:22]: MegaLinter requires you to put C style headers, i.e <...>, before any other so put them above line 22

**Comment by zchochul** [comment]
Thank you for your help @victor-gonzalez!

**Review by victor-gonzalez — APPROVED** [review]

**Pull request merged**